src/services/ui/src/utils/Poller/seaStatusPoller.ts

```diff
@@ -0,0 +1,39 @@
+import { getItem } from "@/api/useGetItem";
+import { DataPoller } from "./DataPoller";
+import { PackageCheck } from "shared-utils";
+
+/** These are checks that simply can't be added into the packagecheck logic.
+ * A prime example being the ability to check for the existance of the data
+ * coming back. This won't work in something like PackageCheck because it is
+ * already expecting data
+ */
+export const pollerSpecificChecks = (
+  data: Awaited<ReturnType<typeof getItem>> | undefined | null,
+) => {
+  const recordExists = !!data;
+
+  return {
+    recordExists,
+  };
+};
+
+export type CheckStatusFunction = (
```

Comment:
status nomenclature should be generalized to include all ideas of data checking, not just statuses.